homeassistant/components/envisalink/alarm_control_panel.py

```diff
@@ -44,6 +48,21 @@
 
     async_add_entities(devices)
 
+    @callback
+    def alarm_arm_max_handler(service):
+        """Map arm max service to method on the Alarm."""
+        entity_ids = service.data.get(ATTR_ENTITY_ID)
+        code = service.data.get(ATTR_CODE)
+        target_devices = [device for device in devices
+                          if device.entity_id in entity_ids]
```

Comment:
`entity_ids` may be `None` as it is not required per the schema.  Consider creating a new schema where it's required or handle when it is `None`

Reply:
Okay sure thanks- makes sense.  Since I'm really expecting the entity_id to be given to me, I've gone ahead and created a different schema for this where i'm requiring it.